views/Chart.react.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+
```

Comment:
Only need one newline between imports and the component.

```suggestion

```

Reply:
[2ccf5ec](https://github.com/AMP-SCZ/dpdash/pull/76/commits/2ccf5ece94d7244e0194567ed7f1990e0e1999a4)